frontend/src/pages/SignIn.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react'
+import { Link, useHistory } from 'react-router-dom'
+import { Button } from 'lib/Button'
+import padlocks from '../assets/15586.jpg'
+import styled from 'styled-components/macro'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  @media (min-width: 768px) {
+    flex-direction: row; 
+  }
+`
+const Image = styled.div`
+  width: 100vw;
+  height: 50vh;
+  background-image: url(${padlocks});
+  background-size: cover; 
+  background-position: center center; 
+
+  @media (min-width: 768px) {
+    width: 50vw;
+    height: 100vh;
+  }
+`
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  width: 100vw;
+  height: 50vh;
+  
+  @media (min-width: 768px) {
+    width: 50vw;
+    height: 100vh;
+  }
+`
+const Title = styled.h1`
+  color: #5D5D5D;
+  font-family: 'Varela Round', sans-serif;
+`
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+`
+const Label = styled.label`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+`
+const Message = styled.p`
+  color: #5D5D5D;
+  text-align: center;
+`
+const Credit = styled.a`
+  color: #5D5D5D;
+  font-size: 10px;
+`
+
+export const SignIn = () => {
+  const url = 'https://fridamaria-secret.herokuapp.com/sessions'
+  const history = useHistory()
+  const [signInValues, setSignInValues] = useState({
+    email: '',
+    password: ''
+  })
+  const [error, setError] = useState('')
+
+  const handleSubmit = event => {
+    event.preventDefault()
+
+    fetch(url,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(signInValues)
+      }
+    )
+
+      .then(res => {
+        if (!res.ok) {
+          throw new Error('Unable to sign in.')
+        }
+        res.json().then(data => {
+          if (data.notFound !== true) {
```

Comment:
This is a little hard to read, with the double negatives. Is it possible to write this as a positive statement instead?